examples/pom.xml

```diff
@@ -48,24 +48,23 @@
     </profile>
     <profile>
       <id>release</id>
+      <activation>
+        <property>
+          <name>!quick</name>
```

Comment:
I would not condition release profile with quick property. Imagine if the release manager does `mvn release:prepare -Dquick` it would not be good.
Further more, the user still have to do `mvn clean install -Prelease` to enable all verification right ? I would activate all verifications by default and disable only with `-Dquick`.